config/999-cache.yaml

```diff
@@ -1,32 +0,0 @@
-# Copyright 2018 The Knative Authors
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     https://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-apiVersion: caching.internal.knative.dev/v1alpha1
-kind: Image
-metadata:
-  name: queue-proxy
-  namespace: knative-serving
-spec:
-  # This is the Go import path for the binary that is containerized
-  # and substituted here.
-  image: github.com/knative/serving/cmd/queue
----
-apiVersion: caching.internal.knative.dev/v1alpha1
-kind: Image
-metadata:
-  name: fluentd-sidecar
-  namespace: knative-serving
-spec:
-  # Cache the fluentd sidecar image
-  image: k8s.gcr.io/fluentd-elasticsearch:v2.0.4
```

Comment:
This reference is only here because its injection is coupled with serving today, if we were to decouple this via either "injection" or a DaemonSet then this would go away.

There also seems to be some misunderstanding of how #283 would manifest.  My expectation is that #283 could be achieved by running a controller that keys off of these resources.  We create these resources dynamically for user containers in the Revision controller, but not these sidecars that we inject (this is a subtle optimization to use 1 Image per sidecar instead of N), which means by deleting these you are removing them from the scope of #283.